openstudiocore/src/utilities/core/StringHelpers.cpp

```diff
@@ -273,4 +274,17 @@ std::string formatUnderscore(const std::string& str) {
   return wStr;
 }
 
+std::vector <std::string> splitString(const std::string &string, char delimiter)

```

Comment:
@macumber @mbadams5 @kbenne So, how about this? (I haven't actually tested it too much yet, more looking to see if it is liked/disliked)